common-features/data-binding/descriptors.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Descriptors - FilterDescriptor, SearchFilter, SortDescriptor, GroupDescriptor
+page_title: Descriptors - FilterDescriptor, SearchFilter, SortDescriptor, GroupDescriptor
+description: Discover the FilterDescriptor, SortDescriptor, SearchFilter and GroupDescriptor properties and how to access their values.
+slug: common-features-descriptors
+tags: telerik,blazor,filterdescriptor, sortdescriptor, groupdescriptor, searchfilter
+published: True
+position: 10
+---
+
+
+## Components with Descriptors
```

Comment:
Consider a title that focuses on the descriptors, not on the components. For example, "Data Operation Descriptors".